discoverable-client/src/main/resources/application.yml

```diff
@@ -96,7 +96,8 @@ apiml:
                 gatewayAuthEndpoint: /api/v1/gateway/auth
                 corsEnabled: false
                 lb:
-                    instanceIdHeader: enabled
+                    type: headerRequest
+                    cacheRecordExpirationTimeInHours: 8
```

Comment:
Why is this here?

Reply:
I guess as an example but we can remove it.

Reply:
yep it was just to document them somewhere in the configuration too, but we can remove it.